src/metabase/search/appdb/core.clj

```diff
@@ -150,9 +150,9 @@
       (search.ingestion/populate-index! :search.engine/appdb))))
 
 (defmethod search.engine/reindex! :search.engine/appdb
-  [_ {:keys [in-place?]}]
+  [_ _]
   (search.index/ensure-ready!)
-  (if in-place?
+  (if search.index/*temp-index-tables*
```

Comment:
Why make this change? Would prefer to only perform this hack when explicitly asked for.

Reply:
I had made the change because the earlier version needed was seeing if combining the in-place? and "mock-table" setup for tests would help fix the failing tests. The switch to making it public was also part of that since it ended up needing both core and indexing that needed access to it. 

But in the end, I was able to put it back